gaffer-as-a-service/gaas-jwt-sidecar/src/main/java/uk/gov/gchq/gaffer/gaas/sidecar/auth/JwtUserDetailsService.java

```diff
@@ -26,17 +28,22 @@
 @Service
 public class JwtUserDetailsService implements UserDetailsService {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(JwtUserDetailsService.class);
+
     @Value("${admin.username: default}")
     private String adminUsername;
     @Value("${admin.password: default}")
     private String adminPassword;
 
     @Override
     public UserDetails loadUserByUsername(final String username) throws UsernameNotFoundException {
+        LOGGER.info("Enter load user by username ");
         if (adminUsername.equals(username)) {
+            LOGGER.info("User  found with username: " + username);
```

Comment:
This line has an extra space after "User" and line 40 has an extra space after "username"

Reply:
Changed as per the review comments